packages/dd-trace/src/constants.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   SPAN_SAMPLING_MAX_PER_SECOND: '_dd.span_sampling.max_per_second',
   DATADOG_LAMBDA_EXTENSION_PATH: '/opt/extensions/datadog-agent',
   DECISION_MAKER_KEY: '_dd.p.dm',
+  PROPAGATED_USER_ID: '_dd.p.usr.id',
```

Comment:
What is this variable? It doesn't seem to be used anywhere.

Reply:
Apparently something ASM is supposed to be setting but apparently is not. Our W3C Trace Context RFC requires that we pass it along with other `_dd.p.*` tags. Originally that was hard-coded with this key but now it finds them dynamically with `startsWith(...)`. Need to ask ASM folks why that tag doesn't already exist though.